### ISSUE TITLE
audio component source mimeTypes are non-standard

### ISSUE THREAD

**Issue by shrugs** [cloudinary/cloudinary-react#160]
## Possible Bug, Feature Request, and Discussion

Thanks for the new Audio component! I'd previously written my own, so i'm excited to chuck out duplicate code. I'd like to ask about the various audio formats and their associated mimeTypes. At first glance, the component renders non-standard mimeTypes, which i'm worried will impact behavior. 

When rendering the Audio component, [this code](https://github.com/cloudinary/cloudinary-react/blob/master/src/components/Video/Video.js#L51) uses the `sourceType` to build the `<source>` mimeType, using the format `audio/${sourceType}`. This works, afaik, for all of the video mimetypes, but based on my research will result in non-standard mimetypes for every audio type besides `aac` and `ogg`. Afaik, here are the correct mimetypes for each cloudinary audio type, noticing that only aac and ogg match the `audio/${sourceType}` mimeType.

```
  ['m4a', 'audio/mp4'],
  ['wav', 'audio/vnd.wav'],
  ['mp3', 'audio/mpeg'],
  ['ogg', 'audio/ogg'],
  ['aiff', 'audio/x-aiff'],
  ['aac', 'audio/aac'],
```

I'm worried that the incorrect mimetypes could cause issues in browsers that i'm not able to detect without running something like BrowserStack. Chrome happily understands `audio/mp3` for example, but afaik it's not in the spec (should be `audio/mpeg`), so i'm not comfortable relying on that behavior.

---

Additionally, is there a specific reason to, [by default](https://github.com/cloudinary/cloudinary-react/blob/master/src/components/Audio/Audio.js#L41), only provide the aac, mp3, and ogg audio sources? If that covers a supermajority of browsers, then that's great. otherwise i think it'd be nice to include all of the sources, for full coverage.

**Comment by asisayag2** [comment]
Thanks @shrugs for the feedback. 
We will take a look shortly.

**Comment by maoznir** [comment]
Hi @shrugs,
This is now fixed in version 1.7.0, please let us know if it works for you!
